src/snowcli/cli/core/__init__.py

```diff
@@ -0,0 +1,11 @@
+import typer
+
+from . import (
```

Comment:
I would suggest using absolute import `from snowcli.cli.core.stage import app as stage_app`. Absolute imports behave better when code is refactored and structure is changing. This can be our case